### PR TITLE
Dynamic Mobile Menu.

### PR DESCRIPTION
**Pull request by mailsg**
Hi,

This pull request is raised to implement dynamic mobile menu using basic `JavaScript` and `DOM` manipulation. Below are the functionalities added : 
1. When the user clicks (or taps) the hamburger button, the mobile menu appears.
2. When the user clicks (or taps) the close (X) button, the mobile menu disappears.
3. When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
4. When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

Kindly review the PR and obliged.

Thanks.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+//     behavior: 'smooth'
+//   });
+// });
+
+// contactLink.addEventListener('click', () => {
+//   document.querySelector('#contact_link').scrollIntoView({
+//     behavior: 'smooth'
+//   });
+// });
+
```
Comment (abdulhamiid) [inline index.js]: Please kindly remove the commented code, let's not commit old pieces of code as inline comments. They will make our project look messy. If you need to review a previous version of your code, you can always use `git history`  🤗🛠

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

[OPTIONAAL] - I notice a lot of repeated use of `width:  1575px;` which does not make your website to be responsive on all screen sizes. 👀.

image
image

image

Percentage is one of the most useful units for creating a responsive or fluid layout. kindly consider using %. so that your design fits well on all screen sizes.

Responsive 💯 👍 `width: 100%`

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (mailsg) [reply index.js]: Removed the inline comments.

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @Team, 




Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**